src/routes/generative-ai/+page.svelte

```diff
@@ -1,86 +1,258 @@
-	<meta property="twitter:card" content={imageSquare} />
+	<meta property="twitter:image" content={aibrain} />
+	<meta property="twitter:card" content={aibrain} />
 </svelte:head>
 
 <LandingHero {title} {description} {imgsrc} {imgalt} />
 <div class="container mx-auto px-10 my-10">
-	<h1 class="text-4xl">Stable Diffusion + ONNX Runtime</h1>
-	<p class="pb-4">Use ONNX Runtime to accelerate this popular image generation model.</p>
-	<h2 class="text-3xl pb-4">Benefits</h2>
-	<div class="grid gap-10 grid-cols-1 lg:grid-cols-2 pb-4">
-		<div class="card bg-base-300">
-			<div class="card-body">
-				<h2 class="card-title pb-4">Run Stable Diffusion outside of a Python environment</h2>
-				<div class="card-actions">
-					<a
-						href="https://onnxruntime.ai/docs/tutorials/csharp/stable-diffusion-csharp.html"
-						class="btn-primary btn">Inference Stable Diffusion →</a
-					>
+	<div class="my-8 grid grid-cols-1 md:grid-cols-4 gap-4">
+		<img class="m-auto rounded-xl" src={aibrain} alt="Representing generative AI">
+		<div class="col-span-3 my-auto">
+
+			<h2 class="text-3xl">What is Generative AI?</h2>
+			<p>
+				Generative AI refers to a type of artificial intelligence that creates new content—such as
+				text, images, audio, or code—based on patterns learned from existing data. Unlike
+				traditional AI models, which primarily classify or predict based on given inputs, generative
+				AI models produce entirely new outputs.
+				<br /><br />
+				They accomplish this through advanced techniques like deep learning, often using models such
+				as Transformers and Generative Adversarial Networks (GANs). Examples include AI that generates
+				human-like text responses, creates realistic images from descriptions, or composes music. Generative
+				AI is driving innovation across industries by enabling personalized experiences, automating creative
+				processes, and opening new possibilities for content generation!
+			</p>
+		</div>
+		
+	</div>
+	<div class="divider" />
+	<div class="">
+		<div class="">
+			<h2 class="text-3xl">Generative AI models</h2>
+			
+		</div>
+		<div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 my-8 gap-4">
+			<div class="flex flex-col h-full gap-4">
+				<div class="flex-1">
+					<h3 class="text-2xl">Text Generation Models</h3>
+					<p>
+						Text generation models are AI systems designed to generate human-like text based on
+						prompts. They're used in chatbots, content creation, summarization, and creative
+						writing. Check out our phi-3 and mistral demos below:
+					</p>
+				</div>
+			
+				<div class="grid gap-4 grid-cols-1 md:grid-cols-2 mt-auto">
+					<a href="https://github.com/microsoft/onnxruntime-genai/blob/main/examples/python/phi-3-tutorial.md" class="btn btn-primary btn-block">Phi-3</a>
+					<a href="https://github.com/microsoft/onnxruntime-inference-examples/tree/main/python/models/mistral" class="btn btn-primary btn-block">Mistral</a>
```

Comment:
Are these meant to be links to models or links to tutorials?

Reply:
links to usable models or tutorials - both should be fine as one is an example one is a tutorial